src/js/basis/basis.ts

```diff
@@ -458,15 +456,36 @@ export class Basis {
         return elementsWithLabels;
     }
 
+    /**
+     * Strips any label associated with atomic symbol
+     * Possible labels:
+     *   (1) Fe1, Fe11
+     *   (2) Fe-a, Fe-b, Fe-1, Fe-1a
+     *   (3) Fe_a, Fe_b, Fe_1, Fe_1a
+     * As of Mar 2025, only single digit numerical labels are allowed
+     */
+    static stripLabelToGetElementSymbol = (elementWithLabel: string): string => {
```

Comment:
@VsevolodX - where do you think this function should go to? ElementWithLabel helper class? Or maybe tools/basis?